hardware/plugins/Plugins.cpp

```diff
@@ -833,8 +833,7 @@ namespace Plugins {
 
 	void CPlugin::Notifier(std::string Notifier)
 	{
-		if (m_Notifier)
-			delete m_Notifier;
+		delete m_Notifier;
```

Comment:
Is this safe ? Maybe StopHardware was called before...
Not knowing what the feature of code will bring, It's safer to check if this pointer is set
(If possible, I would also rename this variable to m_pNotifier)